src/components/Settings/Settings.scss

```
@@ -854,6 +854,29 @@
     line-height: 1.5;
   }
 
+  // Managed-account statement about where the recording goes, shown above the
+  // (hidden, for managed) name field. Smaller and dimmer than the intro copy
+  // above so it reads as a footnote, not another instruction.
+  //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)